ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/policy_maps.schema.yml

```diff
@@ -95,3 +95,90 @@ keys:
+                              - str
+                            description: |
+                              Specify lower rate.
+                              Lower rate range in kbps - <8-200000000>.
+                          cir_unit:
+                            type: str
+                            valid_values:
+                              - "bps"
+                              - "kbps"
+                              - "mbps"
+                            default: "bps"
+                          bc:
+                            description: Set committed burst rate.
+                            required: true
+                            type: int
+                            convert_types:
+                              - str
+                          bc_unit:
+                            type: str
+                            valid_values:
+                              - "bytes"
+                              - "kbytes"
+                              - "mbytes"
+                            default: bytes
+                      rate:
+                        type: dict
+                        keys:
+                          lower_rate:
+                            type: int
+                            convert_types:
+                              - str
+                            description: |
+                              Specify lower rate.
+                              Lower rate range in kbps - <8-200000000>.
+                          rate_unit:
+                            type: str
+                            valid_values:
+                              - "bps"
+                              - "kbps"
+                              - "mbps"
+                              - "pps"
+                            default: "bps"
+                          lower_rate_burst_size:
+                            type: int
+                            convert_types:
+                              - str
+                            min: 256
+                            max: 128000000
+                          burst_size_unit:
+                            type: str
+                            valid_values:
+                              - "bytes"
+                              - "kbytes"
+                              - "mbytes"
+                              - "packets"
+                          action:
+                            description: Set action for policed traffic.
+                            type: str
+                            valid_values:
+                              - "dscp"
+                              - "drop-precedence"
+                          dscp_value:
+                            type: str
+                            convert_types:
+                              - int
+                          higher_rate:
+                            type: int
+                            convert_types:
+                              - str
+                            description: Specify higher rate.
```

Comment:
any range?

Reply:
```
s1-leaf1(config-pmap-c-quality-of-service-quality1-class1)#police rate 8000 bps burst-size 10 kbytes rate 8 bps burst-size 10 kbytes 
% Invalid higher rate
% Higher rate range for specified lower rate is <16-200000008> kbps
s1-leaf1(config-pmap-c-quality-of-service-quality1-class1)#police rate 30000 bps burst-size 10 kbytes rate 8 bps burst-size 10 kbytes
% Invalid higher rate
% Higher rate range for specified lower rate is <38-200000030> kbps
s1-leaf1(config-pmap-c-quality-of-service-quality1-class1)#police rate 100 bps burst-size 10 kbytes rate 8 bps burst-size 10 kbytes
% Invalid lower rate value
% Lower rate range <8-200000000> kbps
% Invalid higher rate
% Higher rate range for specified lower rate is <8-200000000> kbps
s1-leaf1(config-pmap-c-quality-of-service-quality1-class1)#police rate 60000 bps burst-size 10 kbytes rate 8 bps burst-size 10 kbytes
% Invalid higher rate
% Higher rate range for specified lower rate is <68-200000060> kbps
```

Reply:
it is variable and `+8` of the lower rate range

Reply:
Sorry, its because of the unit. The range is defined in `bps` same as lower rate.

Reply:
Added the range in description